constantine/math/elliptic/ec_twistededwards_batch_ops.nim

```diff
@@ -115,9 +115,12 @@ func batchInvert*[F](
     dst[i] *= accumulator
     accumulator *= elements[i]
 
-func batchInvert*[F](dst: openArray[F], source: openArray[F]): bool {.inline.} =
+func batchInvert*[F](dst: var openArray[F], source: openArray[F]): bool {.inline.} =
   if dst.len != source.len:
     return false
   let N = dst.len
   batchInvert(dst.asUnchecked(), source.asUnchecked(), N)
   return true
```

Comment:
Instead of returning a ``bool``, this should likely be a ``debug: doAssert`` as it's an internal function and sanitizing input should be done at a higher level